ci_build_stubs.sh

```diff
@@ -18,6 +18,26 @@ STUBS_DIR="$LEAN_BIN_DIR/generated-stubs"
 # If you do this, know that PyPI and TestPyPI require different API tokens
 PYPI_REPO="pypi"
 
+# For our all additional repos we want to include in our stubs
+ADDITIONAL_REPOS_DIR="$LEAN_DIR/ADDITIONAL_STUBS"
+
+# This function essentially loads in additional repo into our Lean directory
+# which will then be included in our generation of stubs
+function prepare_additional_repos {
+    if [ ! -d $ADDITIONAL_REPOS_DIR ]; then
+        mkdir $ADDITIONAL_REPOS_DIR
+    fi
+
+    cd $ADDITIONAL_REPOS_DIR
+    for REPO in ${ADDITIONAL_STUBS_REPOS[@]}; do
+        #Replace github.com with token from environment before checking it out
+        git clone "${repo//github.com/"${QC_GIT_TOKEN}@github.com"}"
```

Comment:
Bash is case-sensitive ;)

```suggestion
    for repo in ${ADDITIONAL_STUBS_REPOS[@]}; do
        #Replace github.com with token from environment before checking it out
        git clone "${repo//github.com/"${QC_GIT_TOKEN}@github.com"}"
```

Reply:
whoops 😁 I forgot to commit that after testing!

ty ty